tools/rna_tools/trna_mod/gatk_wrapper.xml

```diff
@@ -0,0 +1,90 @@
+<tool id="tRNAmodgatk" name="gatk_for_tRNAmod" version="0.1.1">
+  <description>GATK for tRNAmod analysis</description>
+  <requirements>
+	<requirement type="package" version="2.7">python</requirement>
+	<requirement type="package" version="1.4.1">samtools</requirement>
+	<requirement type="package" version="2.9.2">picard</requirement>
+	<requirement type="package" version="8.0.92">java-jdk</requirement>
+  </requirements>
+  <command>
+	<![CDATA[
+			 ##cleanup
+			 rm -f "${reference}.dict";
```

Comment:
Uha, what is this? You are removing datasets from Galaxy database, I don't think this is a good idea :)